src/components/auth-form.js

```diff
@@ -0,0 +1,66 @@
+import {useState} from 'react'
+
+export default function Auth({
+	onSubmit
+}) {
+	const [loading, setLoading] = useState(false)
+	const [data, setData] = useState({
+		email: '',
+		password: ''
+	})
+	const {email, password} = data
+
+	const handleLogin = async (loginData) => {
+		try {
+			setLoading(true)
+			const {error} = await onSubmit(loginData)
+			if (error) throw error
+			if (!password) {
+				console.info('Check your email for the login link!')
+			}
+		} catch (error) {
+			console.log(error)
```

Comment:
Here we should probably set `loading: false`, otherwise the UI is "stuck" with an error.